src/assets/styles/20-tools/_mixins/_a11y.scss

```diff
@@ -27,3 +27,27 @@
   position: absolute;
   width: 1px;
 }
+
+/**
+ * Hide radio buttons and checkboxes visually (for screen-readers).
+ * Allows Dragon users to access the inputs by using standard voice commands,
```

Comment:
Please could you add a reference link (or links if there are several useful resources) for Dragon to this comment block so other developers can easily find out about it